settings.php

```diff
@@ -590,9 +590,9 @@ function convertseconds($argument)
                                                     <tbody>
                                                         <?php foreach ($dhcp_static_leases as $lease) { ?>
                                                         <tr>
-                                                            <td><?php echo $lease["hwaddr"]; ?></td>
-                                                            <td data-order="<?php echo bin2hex(inet_pton($lease["IP"])); ?>"><?php echo $lease["IP"]; ?></td>
-                                                            <td><?php echo $lease["host"]; ?></td>
+                                                            <td id="MAC"><?php echo $lease["hwaddr"]; ?></td>
+                                                            <td id="IP" data-order="<?php echo bin2hex(inet_pton($lease["IP"])); ?>"><?php echo $lease["IP"]; ?></td>
+                                                            <td id="HOST"><?php echo $lease["host"]; ?></td>
```

Comment:
These element IDs will not be unique, since they are generated for every row.

Reply:
True, but that's also the case with the existing IDs in the DHCP Leases section, which I didn't change:

https://github.com/stevejenkins/AdminLTE/blob/9133ed0b8f1fbeeabc1a2f30a6ad2126533b948f/settings.php#L564-L566

While it's technically not 100% valid HTML, it works. I only used the ID method because it allowed me to apply the same CSS to all the columns in both the DHCP leases and Static reservations sections.

However, I could just as easily add custom classes to all six columns and re-do the CSS. Would you prefer that approach?

Reply:
Yes, it would be an improvement to the situation.